src/gui/options.ui

```diff
@@ -406,6 +406,13 @@
                    </widget>
                   </item>
                   <item>
+                   <widget class="QCheckBox" name="checkOpenDestinationFolderOnBalloonClick">
```

Comment:
qBt may not show balloons on Linux (it uses DBus notifications there) → You have to add this option conditionally.